tendrl/monitoring_integration/alert/handlers/cluster/brick_utilization_handler.py

```diff
@@ -0,0 +1,132 @@
+    representive_name = 'bricks_utilization_alert'
+
+    def __init__(self):
+        AlertHandler.__init__(self)
+
+    def format_alert(self, alert_json):
+        alert  = self.parse_alert_metrics(alert_json)
+        try:
+            alert["alert_id"] = None
+            alert["node_id"] = None
+            alert["time_stamp"] = alert_json['NewStateDate']
+            alert["resource"] = self.representive_name
+            alert['alert_type'] = constants.ALERT_TYPE 
+            alert['severity'] = constants.TENDRL_GRAFANA_SEVERITY_MAP[
+                alert_json['State']]
+            alert['significance'] = constants.SIGNIFICANCE_HIGH
+            alert['pid'] = utils.find_grafana_pid()
+            alert['source'] = constants.ALERT_SOURCE
+            alert['tags']['alert_catagory'] = constants.CLUSTER
+            alert['tags']['cluster_name'] = utils.find_cluster_name(
+                alert['tags']['integration_id'])
+            if alert['severity'] == "WARNING":
+                alert['tags']['message']  = (
+                    "Brick utilization of %s in "\
+                    "cluster %s is %s which is above %s"\
+                    " threshold %s" % (
+                        alert['tags']['brick_path'],
+                        alert['tags']['cluster_name'],
+                        alert['current_value'],
+                        alert['severity'],
+                        alert['tags']['warning_max']
+                    )
+                )
+                     
+            elif alert['severity'] == "INFO":
+                alert['tags']['message'] = (
+                    "Brick utilization of %s in "\
+                    "cluster %s is back normal" % (
+                        alert['tags']['brick_path'],
+                        alert['tags']['cluster_name']
+                    )
+                )
+            else:
+                logger.log(
+                    "error",
+                    NS.publisher_id,
+                    {
+                        "message": "Alert %s have unsupported alert"
+                        "severity" % alert_json
+                    }
+                )
+                raise InvalidAlertSeverity 
+            return alert
+        except (KeyError,
+                CalledProcessError,
+                EtcdKeyNotFound,
+                NodeNotFound,
+                InvalidAlertSeverity) as ex:
+            Event(
+                ExceptionMessage(
+                    "error",
+                    NS.publisher_id, 
+                    {
+                        "message": "Error in converting grafana"
+                        "alert into tendrl alert %s" % alert_json,
+                        "exception": ex
+                    }
+                )
+            )
+    
+    def parse_alert_metrics(self, alert_json):
+        """
+        {
+          EvalData: {
+            evalMatches:[{
+              metric: "averageSeries(tendrl.clusters.ab3b125e-4769-4071-
```

Comment:
Cross check this.
This is not the brick utilization
I am not sure whether this should even be used for anything
Ideally the query should be tendrl.clusters.<cluster_id>.nodes.<node_name>.bricks.<brick_path>.utilization.percent-percent_bytes
Cc @cloudbehl @rishubhjain

Reply:
ya this metric is wrong i will change it